wagtail/contrib/wagtailfrontendcache/tests.py

```diff
@@ -143,21 +146,63 @@ def purge(self, url):
         'BACKEND': 'wagtail.contrib.wagtailfrontendcache.tests.MockBackend',
     },
 })
-class TestCachePurging(TestCase):
+class TestCachePurgingFunctions(TestCase):
+
+    fixtures = ['test.json']
+
+    def test_purge_url_from_cache(self):
+        PURGED_URLS[:] = []  # reset PURGED_URLS to the empty list
```

Comment:
Can this be moved to the `setUp` of this test case, to avoid duplication in every test?